map_model/src/objects/intersection.rs

```diff
@@ -152,10 +152,32 @@ impl Intersection {
     }
 
     // TODO Use osm2streets RoadEdge?
+    // This skips the "interior" piece of any loop roads
     pub fn get_road_sides_sorted(&self, map: &Map) -> Vec<RoadSideID> {
+        // TODO For loop roads, osm2streets repeats the roads! Consider upstream fixes.
```

Comment:
@BudgieInWA, we have an interesting thing happening. When we have a road with i1 == i2, `intersection.roads` ends up with a repeat, something like `[r1, r2, r3, r3]`. When we calculate `RoadEdges` for the intersection geometry, we wind up repeating a left and right side for r3! This is almost definitely causing some horrible bugs. I'll try and tackle over there soon.